# Description: Compile our own intermediate RWHV implementation that sits
#  in content between RWHVBase and Oxide, to work around some non-exported
#  API's
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/content/content_browser.gypi b/content/content_browser.gypi
--- a/content/content_browser.gypi
+++ b/content/content_browser.gypi
@@ -1214,18 +1214,18 @@
       'browser/renderer_host/input/timeout_monitor.h',
       'browser/renderer_host/input/touch_action_filter.cc',
       'browser/renderer_host/input/touch_action_filter.h',
       'browser/renderer_host/input/touch_emulator.cc',
       'browser/renderer_host/input/touch_emulator.h',
       'browser/renderer_host/input/touch_emulator_client.h',
       'browser/renderer_host/input/touch_event_queue.cc',
       'browser/renderer_host/input/touch_event_queue.h',
-      'browser/renderer_host/input/touch_selection_controller_client_aura.cc',
-      'browser/renderer_host/input/touch_selection_controller_client_aura.h',
+      #'browser/renderer_host/input/touch_selection_controller_client_aura.cc',
+      #'browser/renderer_host/input/touch_selection_controller_client_aura.h',
       'browser/renderer_host/input/touchpad_tap_suppression_controller.cc',
       'browser/renderer_host/input/touchpad_tap_suppression_controller.h',
       'browser/renderer_host/input/touchscreen_tap_suppression_controller.cc',
       'browser/renderer_host/input/touchscreen_tap_suppression_controller.h',
       'browser/renderer_host/input/ui_touch_selection_helper.cc',
       'browser/renderer_host/input/ui_touch_selection_helper.h',
       'browser/renderer_host/input/web_input_event_builders_android.cc',
       'browser/renderer_host/input/web_input_event_builders_android.h',
@@ -1303,18 +1303,18 @@
       'browser/renderer_host/render_widget_host_delegate.cc',
       'browser/renderer_host/render_widget_host_delegate.h',
       'browser/renderer_host/render_widget_host_impl.cc',
       'browser/renderer_host/render_widget_host_impl.h',
       'browser/renderer_host/render_widget_host_input_event_router.cc',
       'browser/renderer_host/render_widget_host_input_event_router.h',
       'browser/renderer_host/render_widget_host_view_android.cc',
       'browser/renderer_host/render_widget_host_view_android.h',
-      'browser/renderer_host/render_widget_host_view_aura.cc',
-      'browser/renderer_host/render_widget_host_view_aura.h',
+      #'browser/renderer_host/render_widget_host_view_aura.cc',
+      #'browser/renderer_host/render_widget_host_view_aura.h',
       'browser/renderer_host/render_widget_host_view_base.cc',
       'browser/renderer_host/render_widget_host_view_base.h',
       'browser/renderer_host/render_widget_host_view_mac.h',
       'browser/renderer_host/render_widget_host_view_mac.mm',
       'browser/renderer_host/render_widget_host_view_mac_dictionary_helper.h',
       'browser/renderer_host/render_widget_host_view_mac_dictionary_helper.mm',
       'browser/renderer_host/render_widget_host_view_mac_editcommand_helper.h',
       'browser/renderer_host/render_widget_host_view_mac_editcommand_helper.mm',
@@ -1585,16 +1585,18 @@
       'browser/webui/web_ui_message_handler.cc',
       'browser/zygote_host/zygote_host_impl_linux.cc',
       'browser/zygote_host/zygote_host_impl_linux.h',
       'zygote/zygote_linux.cc',
       'zygote/zygote_linux.h',
       'zygote/zygote_main_linux.cc',
       '<(DEPTH)/../../../shared/port/content/browser/power_save_blocker_oxide.cc',
       '<(DEPTH)/../../../shared/port/content/browser/power_save_blocker_oxide.h',
+      '<(DEPTH)/../../../shared/port/content/browser/render_widget_host_view_oxide.cc',
+      '<(DEPTH)/../../../shared/port/content/browser/render_widget_host_view_oxide.h',
       '<(DEPTH)/../../../shared/port/content/browser/web_contents_view_oxide.cc',
       '<(DEPTH)/../../../shared/port/content/browser/web_contents_view_oxide.h',
     ],
     'android_browser_sources': [
       'browser/android/java/gin_java_bound_object.cc',
       'browser/android/java/gin_java_bound_object.h',
       'browser/android/java/gin_java_bound_object_delegate.cc',
       'browser/android/java/gin_java_bound_object_delegate.h',
